CMakeLists.txt

```diff
@@ -37,31 +37,10 @@ set(ARDUINO_PROJECT_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(ARDUINO_EXTRA_CXXFLAGS "")
 
 set(BOARD_TAG   uno)
-set(BAUD_RATE   38400)
+# Baud rate of 115200 roughly translates to 115 char per 10ms. 
+set(BAUD_RATE   115200)
```

Comment:
Baud rate has been changed so that 100 characters can be sent over serial port in 10 ms interval. This is unlikely to cause any visible performance improvement on arduino side but we are not going to loose any data. This MAY reduce the delay between camera frame acquisition and arduino data acquisition.